### ISSUE TITLE
NTR: Fucosylated N-glycans absent in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8414]
New term request Fucosylated N-glycans absent in blood (elthat_180909083125):   UMLS: 4693907

**Comment by pnrobinson** [comment]
@MickeySegal
can you provide the disease context?

**Comment by MickeySegal** [comment]
https://www.omim.org/entry/618005 "Serum of all 3 patients showed complete loss of total core fucosylated N-glycans compared to controls"

**Comment by pnrobinson** [comment]
@MickeySegal 
[Decreased fucosylation of N-linked protein glycosylation HP:0012353](https://hpo.jax.org/app/browse/term/HP:0012353)
